integration-guides/webfleet.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: Bridgestone Webfleet
+---
+
+To connect your WebFleet telematics device integration with PathStack, you will need to be provided with your Account name, a Username, password and API Key for accessing the API. 
+
+
+To find these credentials, please follow the steps below
+
+<Steps>
+
+  <Step title = "Login to your Webfleet account">
+Visit Webfleet Login Bridgestone Webfleet service and log in to your account. Once logged in, navigate to the 'Users' section.
```

Comment:
For this page, instead of quoting 'Users', bold it without quotes. e.g. `**Users**`